lnrpc/routerrpc/router_backend.go

```diff
@@ -800,6 +800,30 @@ func (r *RouterBackend) UnmarshallRoute(rpcroute *lnrpc.Route) (
 	return route, nil
 }
 
+// maybeValidateMPPParams validates that the MPP parameters are compatible with the
+// payment amount. It returns an error if the parameters don't allow the full
+// payment amount to be sent.
+// maybeValidateMPPParams could be enhanced with additional checks
```

Comment:
i dont think we need this last line. Validation functions can always be enhanced - this is implied